apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestDataPathsActivity.java

```diff
@@ -553,27 +610,43 @@ public void runTest() {
             log("max.allowed.jitter = " + MAX_ALLOWED_JITTER);
             log("test.gap.msec = " + mGapMillis);
 
+            runOnUiThread(() -> {
+                mCheckBoxInputPresets.setEnabled(false);
+                mCheckBoxInputDevices.setEnabled(false);
+                mCheckBoxOutputDevices.setEnabled(false);
+            });
+
             mTestResults.clear();
             mDurationSeconds = DURATION_SECONDS;
 
             if (mCheckBoxInputPresets.isChecked()) {
-                runOnUiThread(() -> mCheckBoxInputPresets.setEnabled(false));
```

Comment:
This strange looking code was designed to allow a user to opt out of some sub-tests while the test was running.

Reply:
Thanks for the heads up! I reverted it